examples/pipelines/two-mappers/Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM bionode/bionode-watermill:dev
```

Comment:
I'm not sure why this is showing in diff, just make sure its the exact same as here https://github.com/bionode/bionode-watermill/blob/dev/examples/pipelines/two-mappers/Dockerfile